en/docs/references/pricing.md

```diff
@@ -0,0 +1,229 @@
+- The Webhook Listener,[Aircraft Status Notifier](#aircraftNotifWebhook) on Choreo then makes **1 database call to the [Flight Scheduling Database](#flightSchedulingDatabase)** to update the flight maintenance status. This is counted as 1 step. 
+
+
+- **Assuming the entire execution time for the update process takes less than 500ms**, the update process thereby results in a **total of 2 steps**.
+
+!!! note
+    The above cost calculation is dependent on the above architecture and design of the Webhooks.
+<br/>
+
+### Example 4: Scheduled Job 
+An airline will revise its ticket prices based on the price of jet fuel. This typically happens once a day. The airline uses a scheduled task on Choreo to run this job and to update the jet fuel price once a day.
+
+#### Scenario 
+ 
+Airlines compute ticket prices typically once per day. The ticket prices depend on a number of factors. The price of oil is one such major factor that affects the pricing.  
+
+The [scheduler](#scheduler) on Choreo runs once per day and fetches the oil prices for the day from the [Oil Price API](#oilPriceAPI). It then computes the jet fuel price applying the price calculation formula. The scheduler then updates the daily jet fuel price in the [Air Fare Database](#airFareDatabase).
+
+Having the jet fuel price, the scheduler is able to now calculate the airfare for each route. The Scheduler retrieves the airline routes from the [Airline Routes API](#airlineRoutesAPI). Upon receiving the airline routes, it iterates over each route, computes the airfare, and updates the AirFare database.
+
+#### Glossary 
+
+- <a class="pricing" id="scheduler">Scheduler</a>: A scheduled task created on Choreo that runs every day.
+- <a class="pricing" id="oilPriceAPI">Oil Price API</a>: An external-facing API that returns the crude oil price for the day.
+- <a class="pricing" id="airFareDatabase">Air Fare Database</a>: The database that stores the jet fuel price which is used to compute the airfare for each route.
+- <a class="pricing" id="airlineRoutesAPI">Airline Routes API</a>: An external-facing API that  provides details about the airline routes and a list of airline routes.
+- <a class="pricing" id="airlineRoutesDatabase">Airline Routes Database</a>: The database that stores the airline routes information.
+
+<br/>
+
+#### Update jet fuel price daily
+
+![ Update jet fuel prices daily](../assets/img/references/scheduled-task-example1.png)
+
+Let’s see how we can calculate the steps for the update maintenance process:
+
+- The scheduled task ([scheduler](#scheduler)) on Choreo runs once per day.
+- The scheduler makes **1 API call to the [Oil Price API](#oilPriceAPI)** to  retrieve the daily oil price.  This is counted as **1 step**. 
+- Upon receiving the daily oil price, the Scheduler then applies the formula to calculate the jet fuel price and calculates daily jet fuel price.
```

Comment:
Have a "the" before "daily jet fuel price".

Reply:
done